site/content/en/guides/consumer/compositepkgs/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Get and apply a remote composite package"
+linkTitle: "Composite Package"
+weight: 7
+type: docs
+description: >
+  Fetch a composite package from a remote git repository and apply its contents to
+  a cluster
+---
+
+{{% pageinfo color="warning" %}}
+
+#### Notice: Composite packages feature support is in alpha phase
+
+{{% /pageinfo %}}
+
+_A kpt package is a directory of resource configs with valid Kptfile in it.
```

Comment:
I think we should describe in some more detail here the rules for how subdirectories and packages interact. Maybe put that into separate text box.

Reply:
Added more information.